app/src/main/java/com/g11x/checklistapp/data/Database.java

```diff
@@ -58,6 +65,13 @@ public Uri insert(ContentValues contentValues) {
         long id = db.insert(Database.ImportantInformation.TABLE_NAME, null, contentValues);
         return ContentUris.withAppendedId(contentUri, id);
       }
+
+      @Override
+      public Cursor query(String[] projection, String selection, String[] selectionArgs,
```

Comment:
Thinking out loud: I wonder if there is going to be any table specific logic at all. Right now the only varying parameter to these methods is the `TABLE_NAME`. After we get a couple more `query` methods implemented, we can make that refactor if it looks like nothing else varies.